src/filesystem.cpp

```diff
@@ -1033,7 +1033,7 @@ bool set_cwd(const std::string& dir)
 	return true;
 }
 
-std::string get_exe_dir()
+std::pair<std::string, std::string> get_exe_dir()
```

Comment:
This seems odd. I'd rather add a `get_exe` or `get_exe_path` function. `get_exe_dir` can then just use that function and return only the dir.